src/worker/worker.ts

```diff
@@ -0,0 +1,37 @@
+import { readExcel } from "./read-wordlist";
+import { WorkerWrapper } from "@common/worker-wrapper";
+import {
+  compileModelFromLexicalModelSource,
+  WordListFromArray,
+} from "@predictive-text-studio/lexical-model-compiler";
+import { linkStorageToKmp } from "./link-storage-to-kmp";
+import Storage from "./storage";
+
+export class WorkerWrapperImpl implements WorkerWrapper {
+  private storage: Storage;
+
+  constructor(storage = new Storage()) {
+    this.storage = storage;
+  }
+
+  saveFile = async (name: string, file: File): Promise<ArrayBuffer> => {
+    const wordlist = await readExcel(await file.arrayBuffer());
+    this.storage.saveFile(name, wordlist);
+    return await linkStorageToKmp(this.storage);
+  };
+
+  compileModel = async (): Promise<string> => {
```

Comment:
Same thing here. I'm very confused about the arrow function syntax being used here!

```suggestion
  async compileModel(): Promise<string> {
```